packages/eslint-config-airbnb/rules/react.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   'rules': {
     // Prevent missing displayName in a React component definition
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md
-    'react/display-name': [0, { 'acceptTranspilerName': false }],
+    'react/display-name': [0, { 'ignoreTranspilerName': false }],
```

Comment:
@ljharb @lencioni did I get this right? https://github.com/airbnb/javascript/tree/master/react#declaration